guillotina_linkintegrity/subscribers.py

```diff
@@ -1,17 +1,21 @@
 import os
 
 from guillotina import configure
+from guillotina.component import get_utility
+from guillotina.contrib.catalog.pg import sqlq
```

Comment:
We should not import contrib packages on modules, no ?

Reply:
Let me do some tests. Maybe we just need to make sure in guillotina this is okay and doesn't auto import it.

The the use-case of linkintegrity, pg catalog will always be enabled.